orc8r/cloud/go/services/metricsd/servicers/protected/servicer.go

```diff
@@ -134,39 +87,11 @@ func preprocessCloudMetrics(family *prom_proto.MetricFamily, hostName string) ex
 	}
 	for _, metric := range family.Metric {
 		metric.Label = protos.GetDecodedLabel(metric)
-		addLabel(metric, metrics.CloudHostLabelName, hostName)
+		servicers.AddLabel(metric, metrics.CloudHostLabelName, hostName)
```

Comment:
Nit: move "helpers" like this to their own pkg -- we generally don't want to import logical fns like this from a servicers pkg (is confusing)

Reply:
cool also replicated these helpers in both servicers